PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetPerformance.cxx

```diff
@@ -2232,30 +2293,35 @@ void AliAnalysisTaskEmcalJetPerformance::ComputeBackground()
     Double_t clusPtSumDCalfid = 0;
     Double_t clusPtSumEMCalRC = 0;
     Double_t clusPtSumDCalRC = 0;
-    
+    Bool_t overlap=0;
     // Loop over tracks. Sum the track pT:
     // (1) in the entire TPC, (2) in the EMCal, (3) in the EMCal fiducial volume, (4) in the DCal, (5) in the DCal fiducial volume, (6) in the EMCal random cone, (7) in the DCal random cone
     // Note: Loops over all det-level track containers. For data there should be only one. For embedding, there should be signal+background tracks.
     AliParticleContainer * partCont = 0;
-    AliTLorentzVector track;
```

Comment:
Please be _very_ careful about changing these existing names, in general you should try to avoid that unless necessary. Check all the rest of the code to make sure it has no affect -- I don't have time to verify right now.

Reply:
I carefully checked. It is used only in the subsequent lines below and in the function GetDeltaR(). It is changed consistently everywhere. I changed this because I needed to additionally get the real track object (not just the track vector) to access the ID in the container.

Reply:
Yea okay thanks I just wanted to be sure that it was carefully checked

Reply:
(I realize you needed also the track object, my comment was directed towards the name modification)